griptape/drivers/vector/opensearch_vector_store_driver.py

```diff
@@ -154,7 +154,7 @@ def query(
                 id=hit["_id"],
                 namespace=hit["_source"].get("namespace") if namespace else None,
                 score=hit["_score"],
-                vector=hit["_source"].get("vector") if include_vectors else None,
+                vector=hit["_source"]["vector"] if include_vectors else [],
```

Comment:
Let's instead make `vector` optional in `BaseVectorStoreDriver.QueryResult`. Then we can revert this change.